Templates/E-Commerce/Magento/Magento to Dynamics365 - Orders/long-description.md

```diff
@@ -0,0 +1,31 @@
+This template process retrieves orders from **Magento** and inserts them into **Dynamics365**.
+Using this template, you can synchronize orders from Magento to Dynamics365.
+
+![Template](assets/Magento-to-Dynamics365-Orders.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Magento user should be eligible to obtain an admin authorization token from Magento.
+- The Dynamics365 user should have access to the client ID, client secret, and tenant ID for the Dynamics365 instance.
+- The products in the orders have already been imported to Dynamics365 using the template for that.
+- The desired unit of measurement for the orders must exist in Dynamics365.
+
+# Implementation and Usage Notes
+
+To insert orders into Dynamics365, a unit of measurement has to be defined for the products in the orders. By default, this process uses a unit of measurement called "Primary Unit", but this behavior can be changed to another unit in the corresponding task. In any case, the desired unit of measurement, be it the default Primary Unit or another one, has to exist in Dynamics365.
+
+The products in the orders to be inserted must also be in a Price List on Dynamics365, to correctly determine their price. This template assumes that the products are already in a price list. If you have imported the products from Magento to Dynamics365 using the template, the products should already be in a price list called "Magento Price List".
+
+The orders inserted into Dynamics365 using this template have an ID starting with "MAG" (as in Magento) followed by their ID from Magento, e.g., "MAG11". Similarly, the names of the imported orders begin with "Magento order" followed by their ID from Magento, e.g., "Magento Order 10".
+
+Existing orders are not updated by this template. Whether an order already exists is checked by comparing the IDs of the orders.
```

Comment:
Can you be a bit more descriptive in the usage notes, as in how the process works? Mostly it's now opening the prerequisites a bit further. What information is retrieved, in which order, and which fields are matched to check if an object is in Dynamics365?

Reply:
Yep, sure! I'll change it a bit 😄